maker-core/cvc.md

```diff
@@ -1,5 +1,40 @@
-# TBC
+# Constitutional Voter Committees (CVCs)
 
-{% hint style="info" %}
-Sorry! We haven't been able to publish documentation on this topic yet. Please check back soon.
-{% endhint %}
+## Overview
+
+Constitutional Voter Committees (CVCs) are organized groups of MKR holders who have a shared vision . They form opinions on the direction of Maker Governance by producing and updating Position Documents.  
+
+Each CVC produces Governance Strategy Position documents and Scope Framework Position Documents to update the [Scope Frameworks](https://endgame.makerdao.com/maker-core/list-of-scopes). This set of documents provide general guidelines for how CDs that follow their governance strategy should vote.
+
+CVCs are one type of Voter Committee. CVCs must have at least one member who is a [Constitutional Conserver (CC)](https://endgame.makerdao.com/maker-core/tbc-governance/constitutional-conservers) and a verified MKR holder. They are subject to eligibility requirements.
+
+## Purpose
+
+CVCs exist to represent the interest of MKR holders. They are responsible to creating documentation to inform Maker holders and align governance with the focus areas of the Maker Protocol. Delegates must follow these strategies.
```

Comment:
Not sure we need this paragraph to explain the purpose of CVCs

Reply:
I was not convinced about this paragraph neither. I "forced" myself to find a "why?" after LFW's feedback but it might be unnecessary.